Claudia/src/main/java/com/endava/task/second/Address.java

```diff
@@ -0,0 +1,31 @@
+package com.endava.task.second;
+
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.city = city;
+        this.country = country;
+        this.street = street;
+    }
+public boolean containsChisinau() {
+        boolean containsChisinau = false;
+        if (this.city.equals("Chisinau")){
+        containsChisinau = true; }
+        else
+            containsChisinau = false;
+
+    return containsChisinau;
+}
+    public String getFullAddress() {
+        String fullAddress = this.country + ", " + this.city + "," + this.street;
+        return fullAddress;
+    }
+
+    public void printAddress() {
+        System.out.println(this.country + ", " + this.city + "," + this.street);
```

Comment:
You can also use String.format() to concatenate string :
String.format("%s, %s, %s", this.country, this.city, this.street)
see also: https://dzone.com/articles/java-string-format-examples

Reply:
Done.